### PR TITLE
Prepare using git-crypt for key management.

### PR DESCRIPTION
**Pull request by Carreau**
Install git-crypt on travis, and setup the gitattributes to encrypt
everything that would end in a `.key`

Move to global variable instead of hardcoding the name of the travis key.

Should we also have all secrets in the subfolder ?

### CODE REVIEW

**Comment by yuvipanda** [comment]
I'd also prefer a secrets/** line in the .giattributes and moving all secrets in there, instead of using *.key.

**Comment by Carreau** [comment]
> I'd also prefer a secrets/** line in the .giattributes and moving all secrets in there, instead of using *.key.

Done.

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+secrets/* filter=git-crypt diff=git-crypt
```
Comment (yuvipanda) [inline .gitattributes]: This needs to be secrets/**, otherwise it doesn't count subdirectories!

Comment (Carreau) [reply .gitattributes]: Done.

**Comment by yuvipanda** [comment]
ty, @Carreau! I'll check if this succeeds!

**Pull request merged**